docs-src/php/how-to-connect-to-a-cluster-php.md

```diff
@@ -0,0 +1,51 @@
+---
+id: how-to-connect-to-a-cluster-php
+title: How to connect to a Cluster in PHP
+sidebar_label: Connect to a Cluster
+---
+
+Create an instance of the `$workflowClient` class and use the `create()` method connect to a Temporal Client to the Temporal Cluster.
+
+Specify the target host, `localhost:7223`, parameter as a string and provide the TLS configuration for connecting to a Temporal Cluster.
+
+```php
+use Temporal\Client\GRPC\ServiceClient;
+use Temporal\Client\WorkflowOptions;
+# . . .
+$workflowClient = Temporal\Client\WorkflowClient::create(
+     ServiceClient::createSSL(
+         'localhost:7233',
+         'certs/ca.cert',
+         'certs/client.key',
+         'certs/client.pem',
+         'tls-sample',
```

Comment:
Is this the namespace parameter?
If so - `your-namespace` - because otherwise no idea what that is for.

Reply:
Looks like this is server name override based on the env var names in the PHP sample.